src/atomdata.cpp

```diff
@@ -55,6 +55,11 @@ namespace Faunus {
             a.id()     = val.value("id", a.id());
             a.mu       = val.value("mu", a.mu);
             a.mulen    = val.value("mulen", a.mulen);
```

Comment:
add to documentation (docs/topology.md)

Reply:
`mu` should be a unit vector and defaults to [1,0,0] so it's always given and there's no need to check. `mulen` defaults to zero.